hrqb/tasks/employee_salary_history.py

```diff
@@ -144,13 +151,51 @@ def _set_base_salary_change_percent(self, salary_df: pd.DataFrame) -> pd.DataFra
                 / new_salary_df["previous_base_amount"]
                 - 1.0
             ),
-            3,
+            PERCENT_DECIMAL_ACCURACY,
         )
         new_salary_df["base_change_percent"] = new_salary_df["base_change_percent"].where(
             new_salary_df["previous_base_amount"].notna(), 0.0
         )
         return new_salary_df
 
+    def _set_effective_salary_and_change_percent(
+        self, salary_df: pd.DataFrame
+    ) -> pd.DataFrame:
+        new_salary_df = salary_df.copy()
+
+        # set effective salary to a temp base, or use the original base amount
+        new_salary_df["effective_salary"] = (
+            new_salary_df["temp_change_base_amount"]
+            .replace(0, np.nan)
```

Comment:
Is the following true: If `new_salary_df["temp_change_base_amount"] == 0`, then `original base amount`?

Reply:
That's correct!